DependencyInjection/DoctrineMigrationsExtension.php

```diff
@@ -32,13 +34,18 @@ class DoctrineMigrationsExtension extends Extension
      */
     public function load(array $configs, ContainerBuilder $container)
     {
-        $configuration = new Configuration();
+        $configuration = $this->getConfiguration($configs, $container);
 
         $config = $this->processConfiguration($configuration, $configs);
 
         foreach ($config as $key => $value) {
             $container->setParameter($this->getAlias().'.'.$key, $value);
         }
+
+        if ($config['profiler_enabled']) {
+            $loader = new Loader\XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
+            $loader->load('services.xml');
```

Comment:
given that this file contains only services related to the profiling (and must stay this was as it is loaded only when profiling is enabled), please name it ``profiler.xml`` or something like that. Otherwise, contributors may add other unrelated services in it in the future (and not notice the mistake as they will develop in an environment with the profiling enabled)